src/app/components/resolve-request-page/resolve-request-page.component.ts

```diff
@@ -149,7 +149,7 @@ submit() {
   }
   this.getIDsFromSubjects(this.selectedSubjects);
   const filter: Filter = new Filter(
-    this.title, format, this.moduleIDs
+    this.title, [format], this.moduleIDs
```

Comment:
Filter was changed to be an array for formats, which is fine. However, in most pages, selFormat is also changed to be an array. We should have consistency, so ideally, selFormat should also be an array. If this isn't possible, then a comment somewhere should explain why.